### DISCUSSION TITLE
Is it possible to temporarily disable TOTP auth factor?

### DISCUSSION THREAD

**Discussion by ashep** [zitadel/zitadel#8224] [Q&A] [answered]
Hello!

Can the TOTP auth factor be temporarily disabled instead of completely removed? Is there an API call for that?

Thank you.

**Comment by hifabienne** [comment]
On the login policy is a list of multifactord, if you have totp in there, the users are able to setup totp.
If you remove it from the list, the users will not be able to use it, but you can add it again later on. 
Is that what you are looking for? https://zitadel.com/docs/apis/resources/mgmt/management-service-update-custom-login-policy

**Reply by ashep** [reply]
No, I meant the ability to enable/disable the TOTP auth factor for any user individually. For example, an admin might manually allow a particular user to skip the TOTP check (disable TOTP) and then enable it again. Currently, I only see the API for TOTP removal, but in this case, the user has to set up it again by themselves.

**Reply by hifabienne** [reply, marked as answer]
ah I see, no this is currently not possible
